data/ha/hawk_test_results.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""Define classes and functions to handle results in HAWK GUI test"""
+
+import time, json, hawk_test_driver, hawk_test_ssh
+
+class resultSetError(Exception):
+    """Base class for exceptions in this module."""
+    def __init__(self, value):
+        self.value = value
+
+    def __str__(self):
+        return repr(self.value)
+
+class resultSet:
+    def __init__(self, test_version='12-SP2'):
+        self.test_version = str(test_version)
```

Comment:
Actually, looking further at that code, the test version is no longer needed in `hawk_test_results.resultSet`. It was required before as the number of tests could vary in the combination of older versions of hawk (12-SP*) and Firefox, but since now each test uses its own instance of the webdriver, whatever caching issue was affecting the `test_generate_report` in 12-SP* under Firefox, is no longer there. Will remove the variable.